src/background.js

```diff
@@ -169,34 +168,28 @@ var pb = {
   * @param {String} origin the third party origin to take action on
   */
   saveAction: function(userAction, origin) {
-    var allUserActions = {'block': pb.USER_BLOCK,
-                          'cookieblock': pb.USER_COOKIE_BLOCK,
-                          'allow': pb.USER_ALLOW};
-    pb.storage.setupUserAction(origin, allUserActions[userAction]);
-    pb.log("Finished saving action " + userAction + " for " + origin);
+    var allUserActions = {'block': this.USER_BLOCK,
```

Comment:
Should this use the *constants* variable instead?

    var allUserActions = {
      'block': constants.USER_BLOCK,
      'cookieblock': constants.USER_COOKIE_BLOCK,
      'allow': constants.USER_ALLOW
    };